### PR TITLE
Frontend 2

### PR DESCRIPTION
**Pull request by hmontesdeoca**

### CODE REVIEW

File: client/src/components/FormPage.jsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            </Form.Group>
+            <Form.Group className="w-50">
+                <Form.Label>Biography</Form.Label>
+                <Form.Control
+                    placeholder="Tell us about yourself!"
+                    as="textarea"
+                    rows={3} />
+            </Form.Group>
+        </Form>
+>>>>>>> 83c53a0fe5dee1e39388c6a8ee72ec2565713e00
```
Comment (LamLieu) [inline client/src/components/FormPage.jsx:75]: Do we need this anymore? @davidlao99 @hmontesdeoca

**Review by LamLieu — CHANGES_REQUESTED** [review]

File: client/src/components/FormPage.jsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
+import { Form, Col} from 'react-bootstrap'
+import React from 'react'
+
+<<<<<<< HEAD
```
Comment (LamLieu) [inline client/src/components/FormPage.jsx:4]: Delete

File: client/src/components/FormPage.jsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+            address2: "",
+            city: "",
+            state: "",
+            zipcode: "",
+            phone: "",
+            email: "",
+            description: ""
+        }
+    }
+=======
```
Comment (LamLieu) [inline client/src/components/FormPage.jsx:21]: Delete

**Review by LamLieu — CHANGES_REQUESTED** [review]

File: client/src/components/FormPage.jsx (hunk 3 above)
Comment (LamLieu) [inline client/src/components/FormPage.jsx]: Delete `=======`

File: client/src/components/FormPage.jsx (hunk 2 above)
Comment (LamLieu) [inline client/src/components/FormPage.jsx]: Delete `<<<<<<< HEAD`

**Review by LamLieu — CHANGES_REQUESTED** [review]

File: client/src/components/FormPage.jsx (hunk 2 above)
Comment (hmontesdeoca) [reply client/src/components/FormPage.jsx]: it has been resolved

**Pull request merged**